app/templates/components/edit-reminder.hbs

```diff
@@ -13,7 +13,8 @@
       {{textarea class='new-reminder-notes' type='text' value=model.notes cols='80' rows='6'}}
     </label>
     {{#link-to 'reminders.reminder' model.id}}
-      <button type='submit' class='save-updates-btn'>Save</button>
+      <button class='save-updates-btn' {{action 'updateReminder' on='click'}}>Save</button>
     {{/link-to}}
+    <button class='undo-updates-btn' {{action 'undoChanges' on='click'}}>Undo</button>
```

Comment:
Also please add a conditional that only makes this button active/visible if there are dirty attributes